### DISCUSSION TITLE
PHP requirement

### DISCUSSION THREAD

**Discussion by pujan123** [PHP-CS-Fixer/PHP-CS-Fixer#6850] [Q&A] [answered]
After git commit command the following message is shown repeatedly "PHP needs to be a minimum version of PHP 5.6.0 and maximum version of PHP 7.4.*". But I am using PHP 8.1.12. Does it mean I need PHP version between 5.6.0 to 7.4.* to run "friendsofphp/php-cs-fixer"?

**Comment by Wirone** [comment, marked as answer]
@pujan123 PHPCSFixer supports PHP versions up to 8.2, but you must need to use at least v3.15 for that. You probably have used some older version and that's why you see such error.

**Reply by pujan123** [reply]
@Wirone "friendsofphp/php-cs-fixer": "^3.4" this is the version I am using currently.

**Reply by Wirone** [reply]
`^3.4` does not actually represent what you use, it's a minimum requirement. Check `composer info friendsofphp/php-cs-fixer` to see installed version.

**Reply by pujan123** [reply]
@Wirone thanks for the help. I got it.
